### DISCUSSION TITLE
What's next for Copilot

### DISCUSSION THREAD

**Discussion by CiberNin** [community/community#31096] [Copilot Conversations]
The code suggestions are nice, they save a bit of typing and searching every here and there. But I feel like the tech could do more. IDK, maybe a side window that generates questions about your code? 

Anyone else have some fun ideas?

**Comment by SaptarshiSarkar12** [comment]
I think that is not a good idea.

**Comment by MulleDK19** [comment]
They have that for VS Code. It was part of Copilot but they made it separate. And it won't come to VS.

**Comment by iamcjwilliams** [comment]
Thank you for your Suggestion! @CiberNin Our Copilot team is working to improve the quality of the User experience!
